pkg/cluster/internal/providers/podman/provision.go

```diff
@@ -179,8 +179,8 @@ func runArgsForNode(node *config.Node, clusterIPFamily config.ClusterIPFamily, n
 		// including some ones podman would otherwise do by default.
 		// for now this is what we want. in the future we may revisit this.
 		"--privileged",
+		"--cgroupns=private",
```

Comment:
is this a breaking change?
https://github.com/containers/podman/issues/4363

Reply:
thanks for pointing it out. I think it is better to drop it.  On cgroup v2 it is the default configuration anyway.